dotnet/src/webdriver/SeleniumManager.cs

```diff
@@ -214,6 +186,27 @@ public static string DriverPath(DriverOptions options)
                 throw new WebDriverException($"Error deserializing Selenium Manager's response: {output}", ex);
             }
 
+            if (result.ContainsKey("logs"))
```

Comment:
`result` object can be strongly typed, just deserialize to object from the json, later it is comfortable to work with this object.

Reply:
Isn't this what I did in the following line?
```
Dictionary<string, string> logs = result["logs"] as Dictionary<string, string>;
```

Reply:
Maybe I do not understand what you are suggesting.